envs/environment_a.yml

```diff
@@ -2,6 +2,7 @@
 name: improver_a
 channels:
   - conda-forge
+  - main
```

Comment:
conda included 'main' implicitly and it must be specified here to continue resolving the environment now with micromaba.